src/Symfony/Component/Mime/Tests/AbstractMimeTypeGuesserTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mime\MimeTypeGuesserInterface;
+
+abstract class AbstractMimeTypeGuesserTest extends TestCase
+{
+    public static function tearDownAfterClass()
+    {
+        $path = __DIR__.'/Fixtures/to_delete';
+        if (file_exists($path)) {
+            @chmod($path, 0666);
+            @unlink($path);
+        }
+    }
+
+    abstract protected function getGuesser(): MimeTypeGuesserInterface;
+
+    public function testGuessImageWithoutExtension()
+    {
+        if (!$this->getGuesser()->isGuesserSupported()) {
+            $this->markTestSkipped('There is nobody to watch the baby!');
```

Comment:
Could we please consider replacing the message inside `markTestSkipped()` by something more useful?

Reply:
oops, copy/paste from Google, fixed.